### ISSUE TITLE
About page should be revisited 

### ISSUE THREAD

**Issue by mairaw** [dotnet-foundation/website#8]
URL: https://dotnetfoundation.org/about

People's titles and their description seem a bit outdated. e.g. both Jon and Martin are listed as .NET Foundation VPs, among many other descriptions that seem too old

**Comment by clairernovotny** [comment]
Can you check the staging site at https://staging.dotnetfoundation.org? That's the current codebase and will hopefully go live next week sometime.

**Comment by mairaw** [comment]
Yes, same problem at https://staging.dotnetfoundation.org/about/board-of-directors

e.g. Martin is no longer at Microsoft and I believe he's no longer the VP correct?
Also @danroth27's description talking about ASP.NET Framework or @terrajobst about .NET Framework seems slightly old. 😉

**Comment by clairernovotny** [comment]
Thanks. PR's are welcome ;) Every page has an "Edit this page" link at the bottom that takes you to the content.

**Comment by mairaw** [comment]
Sure, but I don't know who's VP and these folks want to say on their profile. Maybe each one could update their own profile.

**Comment by danroth27** [comment]
> Every page has an "Edit this page" link at the bottom that takes you to the content.

I'm not seeing an "Edit this page" link on https://dotnetfoundation.org/about. But it's easy enough to find the code.

**Comment by clairernovotny** [comment]
@danroth27 make sure you’re looking at staging.dotnetfoundation.org

**Comment by shawnwildermuth** [comment]
Is this closed?
